apps/prairielearn/elements/pl-graph/pl-graph.py

```diff
@@ -18,6 +19,7 @@
 NEGATIVE_WEIGHTS_DEFAULT = False
 DIRECTED_DEFAULT = True
 LOG_WARNINGS_DEFAULT = True
+DOCTYPE_DECLARATION = r"<!DOCTYPE svg [^>]*>"
```

Comment:
You shouldn't include the doctype for svg elements. This caused the axe accessibility checker to choke.